src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -15,6 +15,7 @@
 import seedu.address.logic.commands.EditCommand;
 import seedu.address.logic.commands.ExitCommand;
 import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.FavouriteCommand;
```

Comment:
Just a small checkstyle error: This import line should be before the import line for FindCommand